### ISSUE TITLE
Split Water Systems end use into interior and exterior like equipment

### ISSUE THREAD

**Issue by MatthewSteen** [NREL/EnergyPlus#6742]
New feature request to split the existing "Water Systems" end use into `Interior Water Systems` and `Exterior Water Systems` as major end use category - i.e. the `End Uses` table, etc. - like interior/exterior equipment is.

**Comment by jmarrec** [comment]
@MatthewSteen Have you put any thoughts into which usage should end up labeled as Interior versus Exterior? I assume your idea is to base the split based on where the point of use is, is that right?

The list of things to put on the Exterior is the smallest, so I'll talk about this one.
`Exterior:WaterEquipment` is an obvious/natural candidate. But what about the others? Here are a couple of questions I can think of:

* Is the Water Usage of a CoolingTower supposed to be exterior? I'd tend to say yes from a water point of view. That being said, if you look at [Group – Exterior Energy Use Equipment](https://bigladdersoftware.com/epx/docs/8-8/input-output-reference/group-exterior-energy-use-equipment.html#group-exterior-energy-use-equipment) mentions "To facilitate the reporting of exterior consumption **to the building’s heat balance/loads considerations**".

* What about a WaterHeater:Mixed, with an Ambient Temperature Indicator set to Outdoors (meaning the unit is outdoor, but the SHW usage is indoors)? (I'd say interior for sure personally)

**Comment by MatthewSteen** [comment]
No, but I think separating interior vs. exterior water use based on whether or not it's used for **the building's heat balance/loads** makes sense for the objects that could go either way, which primarily would be HVACR. Here's a draft proposal.

| Object | Interior Water Systems | Exterior Water Systems | Notes |
| -- | -- | -- | -- |
| Chiller:* | :heavy_check_mark: |  | evaporatively-cooled condensers |
| CoolingTower:* | :heavy_check_mark: |  |  |
| EvaporativeCooler:* | :heavy_check_mark: |  |  |
| EvaporativeFluidCooler:* | :heavy_check_mark: |  |  |
| Exterior:WaterEquipment | :x: | :heavy_check_mark: |
| FluidCooler:* | :heavy_check_mark: |  |  |
| HotWaterEquipment | :heavy_check_mark: |  |  |
| Humidifier:* | :heavy_check_mark:  | :x: |  |
| Refrigeration:Condenser:EvaporativeCooled | :heavy_check_mark: | :x: |  |
| RoofIrrigation | :x: | :heavy_check_mark: |
| SwimmingPool:Indoor | :heavy_check_mark: | :x: |  |
| WaterHeater:* | :heavy_check_mark: | :x: |
| WaterUse:Equipment | :heavy_check_mark: |  |

**Comment by jmarrec** [comment]
Personally I would find it more helpful to report service water (potable cold/hot water used indoor) versus process water and irrigation.
Hence why I thought cooling towers (and other evaporative coolers) would be “exterior”.
Mostly it’s because I could quickly use the numbers to see if I have about the right water consumption at the indoor faucets.

**Comment by MatthewSteen** [comment]
Yes, I agree that makes sense, but for some HVAC classes it starts to become less clear when an object can either be interior or exterior to the building. For example, a direct evaporative section of an AirLoop could either be interior or exterior to the building (same for a humidifier). That's why I think the existing **heat balance/loads**-based definition would work well at a high level.

If water use is instead allocated by service or process use, then it's unclear which category exterior irrigation would go in. 

So the options, would be to split water use by either:

(A) Interior vs. Exterior Water Use

* Water use is allocated to either category based on where it's used except for HVACR classes, which is allocated to interior water use because it is used for the building's heat balance/loads.

(B) Service vs. Process Water Use

* Where would exterior irrigation go?

**Comment by MatthewSteen** [comment]
Thinking about this more, I think that *(re)moving* the `Water` column from the `End Uses` table and putting that information in a reformatted `Water Source Summary` table could make more sense. This would separate *energy* end uses from *water* end uses, avoid mixed energy/water units (i.e. kBtu and gal), and create a nice concise summary of the resources that a building uses by having the following tables.

* Energy End Uses
* Water End Uses
